submissions/Articles/global-payouts-non-profits.md

```diff
@@ -0,0 +1,24 @@
+﻿
+# How Global Payouts Can Empower Communities and Non-profits
+
+You’re hosting a charity baking event, raising funds to help an underserved community overseas gain access to 24/7 electricity and clean water. After reaching your monetary goal, you send the money using a platform like PayPal, feeling accomplished. But a few days later, when you check for updates, there's no clear sign that the funds have reached the community. The excitement of making a difference turns into frustration and uncertainty—did the money get there? Or is it stuck in transit, caught up in fees and delays? A possible solution to this issue is global payouts. They are the transfer of funds between two parties in different countries and/or involving two different currencies. Still uncertain about this method? No worries, this article will explore how global payouts empower your non-profits and community initiatives.
+
+## How Global Payouts Offer Transparency and Save You Money
+
+A common issue that non-profits encounter when using traditional banking to fund money overseas is that most of these platforms require them to pay fees to access to some or all of their services and to make transactions.  For example, [nonprofits spend around $3 billion a year on transaction fees](https://www.zeffy.com/blog/nonprofits-paid-2-billion-in-transaction-fees-last-year). In other words, traditional banking platforms are too expensive.  Global payout platforms can solve this issue by giving nonprofits the freedom the payment-sending rates that fall within your budget and are upfront about their fees.  Chimoney was started by [Uchi Uchibeke after struggling to send winners the prize money for an event they were hosting due to the traditional banking platform fees being expensive](https://chimoney.io/blogs/techstars-backs-chimoney-to-revolutionize-global-payouts/). From there, the winners were paid. Now no hidden fees aren’t the only way global payouts can empower your community initiative or nonprofit. Let’s check out another one.
```

Comment:
Please update this part `an event they were hosting` to `AfriHacks, a hackathon they were hosting`